sample.env

```diff
@@ -1,3 +1,4 @@
 ESLINT_NO_DEV_ERRORS=true
 TSC_COMPILE_ON_ERROR=true
-REACT_APP_UNCHAINED_ETHEREUM_URL=http://localhost:31300/api/v1
+REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=http://localhost:31300
```

Comment:
leave path IMO for clarity

Reply:
i think this was changed elsewhere and it's just the url now